pt-BR/active_storage_overview.md

```diff
@@ -430,27 +430,24 @@ Image analysis provides `width` and `height` attributes. Video analysis provides
 
 Analysis requires the `mini_magick` gem. Video analysis also requires the [FFmpeg](https://www.ffmpeg.org/) library, which you must include separately.
 
-Transforming Images
+Transformando Imagens
 -------------------
 
-To enable variants, add the `image_processing` gem to your `Gemfile`:
+Para ativar variações, adicione a gema  `image_processing`  no seu `Gemfile`:
```

Comment:
```suggestion
Para ativar variações, adicione a _gem_  `image_processing`  no seu `Gemfile`:
```

Eu acho legal essa tradução mas como no resto do guia acho que mantivemos em itálico, fica fora do padrão. Mas podemos rever isso para todas as páginas no futuro 😄